.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -99,6 +99,16 @@ steps:
           .buildkite/scripts/steps/package.sh
         artifact_paths:
           - "build/distributions/**/*"
+  
+  - label: ":elastic-stack: Trigger Elastic Agent msi build"
+    trigger: "elastic-stack-installers"
+    build:
+      branch: main
+      commit: HEAD
```

Comment:
This guarantees that the same commit is used in the MSI pipeline? That is, this is translated into commit hash XYZ which is what the triggered pipeline checks out? 

The triggered pipeline doesn't also check out HEAD? If it did this, there is a chance there was another commit to main between the two pipelines running.

Reply:
MSI always takes the latest snapshots available and use them to create the msi, they do not rely on a commit hash per say.